internal/db/modelmigration_test.go

```diff
@@ -0,0 +1,155 @@
+// Copyright 2025 Canonical.
+
+package db_test
+
+import (
+	"context"
+	"database/sql"
+	"testing"
+
+	qt "github.com/frankban/quicktest"
+
+	"github.com/canonical/jimm/v3/internal/db"
+	"github.com/canonical/jimm/v3/internal/dbmodel"
+	"github.com/canonical/jimm/v3/internal/errors"
+)
+
+func TestAddModelMigrationUnconfiguredDatabase(t *testing.T) {
```

Comment:
We add these all over the place, I don't think we should be duplicating it constantly... Maybe just delete this?